ci-operator/step-registry/openshift-qe/network-policy/openshift-qe-network-policy-ref.yaml

```diff
@@ -0,0 +1,27 @@
+ref:
+  as: openshift-qe-network-policy
+  from_image:
+    namespace: ci
+    name: ocp-qe-perfscale-ci
+    tag: latest
+  commands: openshift-qe-network-policy-commands.sh
+  timeout: 8h0m0s  
+  env:
+  - name: WORKLOAD
+    default: "networkpolicy-case1"
+    documentation: |-
+      Default is "networkpolicy-case1", support senario networkpolicy-case2 and networkpolicy-case3
+  - name: GC
```

Comment:
Since this workload is under the kube-burner folder not kube-burner-ocp-wrapper the cleanup value is CLEANUP_WHEN_FINISH I believe

Reply:
Fixed in new code, thanks!